### ISSUE TITLE
Usage problem with the fastai dependency

### ISSUE THREAD

**Issue by Vincentx15** [kireygroup/fastsom#4]
Hello, I'm trying to follow the basic tutorial to use your tool. Running "pip install fastsom" installs fastai as a dependency without problem in a fresh conda env. However trying to import it then fails :+1: 

```
In [1]: from fastsom import *
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-9129ec1896d3> in <module>
----> 1 from fastsom import *

~/.local/lib/python3.7/site-packages/fastsom/__init__.py in <module>
      3 """
      4 from .core import *
----> 5 from .data_block import *
      6 from .interp import *
      7 from .learn import *

~/.local/lib/python3.7/site-packages/fastsom/data_block/__init__.py in <module>
----> 1 from .databunch import *
      2 from .transform import *
      3 from .utils import *

~/.local/lib/python3.7/site-packages/fastsom/data_block/databunch.py in <module>
      6 import numpy as np
      7 
----> 8 from fastai.tabular import TabularLine, TabularList, TabularProcessor, TabularProc, TabularDataBunch, OrderedDict
      9 
     10 from fastsom.core import ifnone, find

ImportError: cannot import name 'TabularLine' from 'fastai.tabular' (/home/bougui/.local/lib/python3.7/site-packages/fastai/tabular/__init__.py)
```
I would think that the TabularLine object is deprecated, could you help me solving this problem ?

Thanks a lot in advance !
Vincent

**Comment by rsayn** [comment]
Hi, I think this is due to the recent update of Fast.ai to version 2.
I am currently working to update the library as soon as possible.

In the meantime, you can install the old compatible version with `pip install fastai==1.0.60`

**Comment by rsayn** [comment]
This was fixed by setting `fastai==1.0.60` inside `setup.py`. New installs should now work as expected.
